### DISCUSSION TITLE
What is the smallest MIR output that I can create using the c api?

### DISCUSSION THREAD

**Discussion by rishavs** [vnmakarov/mir#336] [Q&A] [answered]
I am trying to smoke test the c api in my toy compiler and wanted to know what is the smallest MIR program I can generate which I can test against.
Something like this in c
```
#include <stdio.h>
int main() {
    return 0;
}
```
I just want to know the I have integrated MIR properly.

**Comment by vnmakarov** [comment, marked as answer]
You can use `int main () {return 0;}` and compile it by `c2m` with option `-S`.  It will produce `.mir` file you can use for your purposes:

```
M0:     module
main:   func    i32

# 0 args, 0 locals, 0 globals
	ret     0
	endfunc
	export  main
	endmodule
```
